### DISCUSSION TITLE
[Android] 코틀린 컨벤션

### DISCUSSION THREAD

**Discussion by 1jeongg** [BDD-CLUB/challenge-every-week-posting#183] [1. 개발 관련 글 모음]
컨벤션은 개발자에게 중요한 부분이죠..
[코틀린 컨벤션](https://1jeongg.tistory.com/10)에 대해 정리해봤습니다!

**Comment by pnudpwls1103** [comment]
항상 이름짓기는 힘들죠..! 그래서 chatgpt로 추천 종종 받는거 같네요 ㅋㅎ
